etl/steps/data/garden/energy/2024-05-08/photovoltaic_cost_and_capacity.py

```diff
@@ -151,11 +151,8 @@ def run(dest_dir: str) -> None:
     # Add column for region.
     tb_combined = tb_combined.assign(**{"country": "World"})
 
-    # Set an appropriate index and sort conveniently.
-    tb_combined = tb_combined.set_index(["country", "year"], verify_integrity=True).sort_index()
-
-    # Rename table.
-    tb_combined.metadata.short_name = paths.short_name
+    # Format table conveniently.
```

Comment:
On line 34, 35: in the future we can maybe adjust by inflation with the data from WDI, as Fiona did [here](https://github.com/owid/etl/blob/master/etl/steps/data/garden/health/2024-04-22/gpei_funding.py )